src/sentry/api/bases/organization.py

```diff
@@ -146,6 +146,10 @@ class OrganizationSearchPermission(OrganizationPermission):
     }
 
 
+class OrganizationDataExportPermission(OrganizationPermission):
+    scope_map = {"GET": ["event:read", "event:admin"], "POST": ["event:read", "event:admin"]}
```

Comment:
I'm not sure, because from how I see it the POST route still just reads existing events. They just start an export to read it in a different way (i.e. as a downloaded CSV)

I might be misunderstanding the scopes though, if I should be changing this please lmk

Reply:
@scefali Explained it, I totally misunderstood, changed in cfc7420